### PR TITLE
docs: update outputs

### PR DESCRIPTION
**Pull request by kassianh**

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -1091,56 +1091,100 @@ section {
...
+        URL that can be provided to git clone to clone the repository via HTTPS.
+      END
+    }
 
-        A merged map of deploy key objects for the keys originally passed via `deploy_keys` and `deploy_keys_computed` as returned by the [`github_repository_deploy_key`] resource keyed by the input `id` of the key.
+    output "git_clone_url" {
+      type        = string
+      description = <<-END
+        URL that can be provided to git clone to clone the repository
+        anonymously via the git protocol.
```
Comment (thiesen) [inline README.tfdoc.hcl:1137]:
```suggestion
        URL that can be provided to git clone to clone the repository anonymously via the git protocol.
```

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -1091,56 +1091,100 @@ section {
...
+      END
+    }
 
-      - **`projects`**
+    output "collaborators" {
+      type        = object(collaborators)
+      description = <<-END
+        A map of Collaborator objects keyed by the `name` of the collaborator as
+        returned by the [`github_repository_collaborator`] resource.
```
Comment (thiesen) [inline README.tfdoc.hcl:1145]:
```suggestion
        A map of Collaborator objects keyed by the `name` of the collaborator as returned by the [`github_repository_collaborator`] resource.
```

File: README.tfdoc.hcl [hunk 3]
```diff
@@ -1091,56 +1091,100 @@ section {
...
+    }
 
-        A map of Project objects keyed by the `id` of the project as returned by the [`github_repository_project`] resource
+    output "deploy_keys" {
+      type        = object(deploy_keys)
+      description = <<-END
+        A merged map of deploy key objects for the keys originally passed via
+        `deploy_keys` and `deploy_keys_computed` as returned by the
+        [`github_repository_deploy_key`] resource keyed by the input `id` of the
+        key.
```
Comment (thiesen) [inline README.tfdoc.hcl:1155]:
```suggestion
        A merged map of deploy key objects for the keys originally passed via `deploy_keys` and `deploy_keys_computed` as returned by the [`github_repository_deploy_key`] resource keyed by the input `id` of the key.
```

File: README.tfdoc.hcl [hunk 4]
```diff
@@ -1091,56 +1091,100 @@ section {
...
+      END
+    }
 
-      - **`issue_labels`**
+    output "projects" {
+      type        = object(project)
+      description = <<-END
+        A map of Project objects keyed by the `id` of the project as returned by
+        the [`github_repository_project`] resource
```
Comment (thiesen) [inline README.tfdoc.hcl:1163]:
```suggestion
        A map of Project objects keyed by the `id` of the project as returned by the [`github_repository_project`] resource
```

File: README.tfdoc.hcl [hunk 5]
```diff
@@ -1091,56 +1091,100 @@ section {
...
-      - **`webhooks`**
-
-        All attributes and arguments as returned by the github_repository_webhook resource.
-
-      - **`secrets`**
+    output "webhooks" {
+      type        = object(webhook)
+      description = <<-END
+        All attributes and arguments as returned by the
+        `github_repository_webhook` resource.
```
Comment (thiesen) [inline README.tfdoc.hcl:1178]:
```suggestion
        All attributes and arguments as returned by the `github_repository_webhook` resource.
```

File: README.tfdoc.hcl [hunk 6]
```diff
@@ -1091,56 +1091,100 @@ section {
...
-
-          URL that can be provided to git clone to clone the repository via HTTPS.
-
-        - **`git_clone_url`**
+    output "repository" {
+      type        = object(repository)
+      description = <<-END
+        All repository attributes as returned by the [`github_repository`]
+        resource containing all arguments as specified above and the other
+        attributes as specified below.
```
Comment (thiesen) [inline README.tfdoc.hcl:1101]:
```suggestion
        All repository attributes as returned by the [`github_repository`] resource containing all arguments as specified above and the other attributes as specified below.
```

Comment (kassianh) [reply README.tfdoc.hcl:1101]: Hmm. I broke these up into multiple lines for easier readibility, especially here on Github. Had a conversation about this with Tiago [here](https://github.com/mineiros-io/terraform-google-cloud-sql-proxy/pull/7#discussion_r786714166). Let me know what you think @thiesen

Comment (thiesen) [reply README.tfdoc.hcl:1101]: got it! my only concern is that I would expect the text to be displayed as defined in the input (even though this the rendered document shows everything in the same line) so this brings me to the question: do you believe this is the correct behaviour or should we iterate on it on Terradoc side so the text we input matches what is rendered?

**Review by thiesen — APPROVED** [review]
lgtm (you can disconsider the suggestions as per https://github.com/mineiros-io/terraform-github-repository/pull/92/files#r793092411 )

**Pull request merged**